signalfx-tracing/signalfx-nodejs-tracing/koa/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "wordExplorer",
+  "version": "1.0.0",
+  "description": "A SignalFx Koa instrumentation example",
+  "main": "index.js",
+  "scripts": {
+    "start": "SIGNALFX_TRACING_DEBUG=true SIGNALFX_SERVICE_NAME=wordExplorerServer nodemon ./server.js",
+    "client": "SIGNALFX_SERVICE_NAME=wordExplorerClient node ./client.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "nodemon": "^1.18.10"
+  },
+  "dependencies": {
+    "dotenv": "^7.0.0",
+    "koa": "^2.7.0",
+    "koa-bodyparser": "^4.2.1",
+    "koa-logger": "^3.2.0",
+    "koa-router": "^7.4.0",
+    "koa-static": "^5.0.0",
+    "mongoose": "^5.4.22",
+    "node-wordnet": "^0.1.12",
+    "signalfx-tracing": "0.0.4",
```

Comment:
`0.0.6` is the most recent, and this should work w/ any later maintenance release: https://docs.npmjs.com/about-semantic-versioning

Reply:
Updated